app/views/epub_ebooks/show.html.erb

```diff
@@ -147,9 +147,6 @@
         })
         <% end %>
 
-        // No modal panel initiate
-        <%= render "cozy_controls_nomodal", presenter: @presenter, parent_presenter: @parent_presenter %>
```

Comment:
@conorom I believe the code in here is still needed...there are some things that could be removed in the file, but I'm pretty certain lines 1-18 of `app/views/epub_ebooks/_cozy_controls_nomodal.html.erb` still need to be executed.